Recipes/Recipes.API/Controllers/RecipesController.cs

```diff
@@ -84,6 +98,70 @@ public async Task<ActionResult> DeleteRecipe(int recipeId)
         return NoContent();
     }
 
+    [HttpPost]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult> CreateRecipe(RecipeDTO request)
+    {
+        try
```

Comment:
Yeah I'm not sure about validation. It would be nice to have some sort of validation layer on all of the incoming DTOs. I don't mean if the json can be parsed , but more so of business validation, e.g. "servings must be >= 1", "preparation time must be above 1 min", etc.

Reply:
Also there's a requirement to have n-tier architecture and we're kind of lacking any "business" layer. From the presentation (REST API) layer we're taking data from Data Access layer and returning it = there's no business stuff. Maybe implementing business validation rules could be the start of it

Reply:
Yeah, we should probably atleast have 3 layers.  I guess we could create a separate issue for a Business layer.